src/common/config.ts

```diff
@@ -8,6 +8,7 @@ export const CONFIG_URL = process.env.REACT_APP_ARVADOS_CONFIG_URL || "/config.j
 
 export interface Config {
     API_HOST: string;
+    KEEP_WEB_HOST: string;
```

Comment:
can we have camelCase fields here?

Reply:
I thought config is a constant so it should not be in camelCase.